bundles/io/org.eclipse.smarthome.io.voice/src/main/java/org/eclipse/smarthome/io/voice/text/AbstractRuleBasedInterpreter.java

```diff
@@ -228,7 +295,7 @@ protected Rule itemRule(Object headExpression) {
     protected Rule itemRule(Object headExpression, Object tailExpression) {
         Expression tail = exp(tailExpression);
         Expression expression = tail == null ? seq(headExpression, name())
-                : seq(headExpression, name(tail.getFirsts()), tail);
+                : seq(headExpression, name(tail), tail);
```

Comment:
It was the sequence of possible stopper tokens. Whenever one of them occurs, one of the right side - "stopping" - expressions would match. However: This is not done like that anymore. Now getFirsts is only used for generating a grammar.